spring/boot2-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaClientHttpConnector.java

```diff
@@ -110,8 +109,10 @@ private ArmeriaClientHttpRequest createRequest(HttpMethod method, URI uri) {
         return new ArmeriaClientHttpRequest(builder.build(), method, pathAndQuery, uri, factoryWrapper);
     }
 
-    private ArmeriaClientHttpResponse createResponse(ResponseHeaders headers,
-                                                     ArmeriaHttpClientResponseSubscriber s) {
-        return new ArmeriaClientHttpResponse(headers, s.toResponseBodyPublisher(), factoryWrapper);
+    private CompletableFuture<ArmeriaClientHttpResponse> createResponse(HttpResponse response) {
+        final ArmeriaHttpResponseBodyStream bodyStream =
+                new ArmeriaHttpResponseBodyStream(response, CommonPools.workerGroup().next());
```

Comment:
Can we use the eventLoop that is used for sending a request?
If not, how about just using `response.defaultSubscriberExecutor()` in `DefaultSplitHttpResponse`?

Reply:
It is difficult to capture or pass RequestContext at this point. Let me use `response.defaultSubscriberExecutor()` :-)